### DISCUSSION TITLE
Update

### DISCUSSION THREAD

**Discussion by am3vin** [community/community#57679] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi cann you pleas update [SC2Expansion] i want to play it on the new update or make an toturial to install it pleas thanks

**Comment by otter-computer** [comment]
Hi there,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the project's repository. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
